core/src/main/scala/com/avast/cloud/metrics/datadog/api/Count.scala

```diff
@@ -0,0 +1,9 @@
+package com.avast.cloud.metrics.datadog.api
+
+import scala.language.higherKinds
+
+trait Count[F[_]] {
```

Comment:
Counter?

Reply:
I called it counter to keep it consistent with datadog metrics api (https://docs.datadoghq.com/developers/metrics/)